components/commons/dropshadows.css

```diff
@@ -0,0 +1,51 @@
+/*!
+Copyright 2023 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+%spectrum-DropShadow-key {
+  --spectrum-drop-shadow-color-key: var(--spectrum-drop-shadow-color);
```

Comment:
Would it be valuable to split out the variable definitions from the css property definitions? Just trying to think of examples where someone might want to variables and properties to source from different levels (properties at root for example).